### PR TITLE
Updated DocBlocks to correct template override location (tribe vs tri…

### PR DESCRIPTION
**Pull request by sdokus**
…be-events)

### 🎫 Ticket

[GTRIA-1170] 

### 🗒️ Description

Updated the template paths in the DocBlocks for all the files in `src/views/tickets`. The path was previously `[your-theme]/tribe-events/[...]` but it should be `[your-theme]/tribe/[...]`

### ✔️ Checklist
- [X] I've included a changelog entry in the readme.txt file. 
- [X] My code is tested. 
- [X] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[GTRIA-1170]: https://stellarwp.atlassian.net/browse/GTRIA-1170?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by juanfra — CHANGES_REQUESTED** [review]
Thanks @sdokus!

Whenever you're updating templates, you'll need to set the `@version TBD` and `@since TBD Description of the changes you have done`.

That way the system makes the interpretation of the latest version and warns users of templates with changes in the `Recent template changes` of the `Troubleshooting` page.

**Review by bordoni — DISMISSED** [review]

File: readme.txt [hunk 1]
```diff
@@ -194,6 +194,8 @@ We've got a [LoopedIn](https://evnt.is/ideas) page where we're actively watching
 Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on using, tweaking, and troubleshooting our plugins.
 
 == Changelog ==
+= [TDB] =
```
Comment (juanfra) [inline readme.txt:197]:
```suggestion

= [TBD] TBD =

```

File: readme.txt [hunk 2]
```diff
@@ -194,6 +194,8 @@ We've got a [LoopedIn](https://evnt.is/ideas) page where we're actively watching
...
+* Tweak - Changed incorrect file paths in DocBlocks for template overrides for all files in src/views/tickets. [GTRIA-1170]
```
Comment (juanfra) [inline readme.txt:198]:
```suggestion
* Tweak - Changed incorrect file paths in DocBlocks for template overrides for all files in `src/views/tickets`. [ET-2004]
```

**Review by juanfra — CHANGES_REQUESTED** [review]
Thanks for working on this, Sam! I left a few minor comments for the readme and we are good to go

**Comment by redscar** [comment]
This looks good. I would update the branch to point to G24.boxer and you should be good to go.

**Review by codingmusician — APPROVED** [review]

**Review by juanfra — APPROVED** [review]

**Pull request merged**